### PR TITLE
zk-exhibitor: startup after update-os is active, README updates

### PR DESCRIPTION
**Pull request by chr0n1x**

### CODE REVIEW

**Comment by iderdik** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -63,16 +44,74 @@ From there, it does a couple of things:
...
+All secrets & key management is a bit adhoc. Most of the `setup` scripts, which house the logic for setting up the data for then fleet units to use, require a few things to download secrets & keys:
+
+- the `$CONTROL_TIER_S3SECURE_BUCKET` environment variable, written into `/etc/environment` by cloudformation
+- [behance/docker-aws-s3-downloader](https://github.com/behance/docker-aws-s3-downloader) container to download files
+- IAM roles to access `$CONTROL_TIER_S3SECURE_BUCKET`
+
+Secrets make it onto the nodes in the form of flat text files that live within `$CONTROL_TIER_S3SECURE_BUCKET`. The `setup` files **individually** know which file(s) they need to download & how to read, set or use the data for their corresponding units. So for example, the [datadog unit](https://github.com/adobe-platform/mesos-systemd/blob/master/v2/fleet/datadog.service#L21) requires an `etcd` key, `/ddapikey`. Knowing this, we have a [datadog setup script](https://github.com/adobe-platform/mesos-systemd/blob/master/v2/setup/datadog.sh) which downloads a `.datadog` file from `$CONTROL_TIER_S3SECURE_BUCKET`, expects it to be in a certain format, and sets the etcd key.
+
+#### Files in S3 (deprecated)
```
Comment (iderdik) [inline README.md]: Can you add what we plan to replace it with? (exploring KMS-encrypted dynamo table). Deprecated implies they are no longer necessary which is not quite true. This is more of a "plan to deprecate"

File: README.md [hunk 2]
```diff
@@ -63,16 +44,74 @@ From there, it does a couple of things:
...
+
+##### Services, dotfiles, dotfile formats
+
+| Service       | File | Format |
+| ------------- | ------------- | ------------- |
+| Datadog | `.datadog` | Just the key. Nothing else. |
+| Sysdig | `.sysdig` | Just the key. Nothing else. |
+| Sumologic | `.sumologic` | `ID=YOURID`<br/>`SECRET=YOURSECRET`|
+| Flight Director | `.flight-director` | `/FD/GITHUB_CLIENT_ID (YOUR GITHUB APP ID)`<br/>`/FD/GITHUB_CLIENT_SECRET (YOUR GITHUB APP SECRET)`<br/>`/FD/GITHUB_ALLOWED_TEAMS org/team` |
+| HUD | `.hud` | `/HUD/client-id (GITHUB_APP_ID can == flight director)`<br/>`/HUD/client-secret (GITHUB_APP_SECRET can == flight director)`|
```
Comment (iderdik) [inline README.md]: Nitpick... this reads like the value can be "flight director". Consider something like can be the same as the one used in .flight-director.

File: README.md (hunk 1 above)
Comment (matthewdfuller) [inline README.md]: This is a beautiful chart that will help everyone out so much

**Comment by nBerg** [comment]
:+1:

**Comment by iderdik** [comment]
:+1:

**Pull request merged**